apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/condition/request/ReadConditionRequest.java

```diff
@@ -0,0 +1,24 @@
+package gov.cdc.nbs.questionbank.condition.request;
+
+import lombok.Data;
+
+@Data
+public class ReadConditionRequest {
+
+    private String id;
+
+    private String conditionShortNm;
+
+    private String progAreaCd;
+
+    private String familyCd;
+
+    private String coinfectionGrpCd;
+
+    private Character nndInd;
+
+    private String investigationFormCd;
+
+    private Character statusCd;
+
+}
```

Comment:
What will the UI look like for this search? Will it be a table with a search box at the top? If so, it should the request should have a single input parameter. Usually named "searchText" or similar.

Reply:
Hmm looking at: 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/c90ca67d-7a68-47ce-8b8a-307e733a9efe)

It seems we will need to support both a generic search input as well as individual field filtering where if both are supplied they would be `AND`ed together

So we may need to change the input for the search to be a single text entry as well as a list of filters.